templates/violation/detail.html

```diff
@@ -12,36 +12,36 @@
 
 {% block header %}
     <h2 class="page-title">
-        <i class="fa fa-fw fa-exclamation-triangle"></i> Incident
+        <i class="fa fa-fw fa-exclamation-triangle"></i> {% trans "Incident" %}
         {% if violation.geoname %}
-            in <strong>{{ violation.geoname }}</strong>
+            {% trans "in" %} <strong>{{ violation.geoname }}</strong>
```

Comment:
We might want comments for these isolated prepositions letting the translator know what the different kinds of output of the string are supposed to look like:

`Incident on <date>`
`Incident between <date> and <date>`
`Incident starting <date>`
`Incident ending <date>`

Reply:
Nice thought!